### PR TITLE
Add starter tests for WOF placetypes

### PR DESCRIPTION
**Pull request by trescube**
- counties
- countries
- localadmins
- localities
- macrocounties
- macroregions
- neighbourhoods
- regions

Tests were chosen to satisfy the following conditions:

- diacriticals
- punctuation
- abbreviations
- multiple tokens
- simple hierarchies
- complex hierarchies
- ambiguity
- known issues (such as Puerto Rico being treated as a region when it's a dependency)

closes pelias/pelias#218

### CODE REVIEW

File: test_cases/wof_countries.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
+{
+  "name": "Who's on First counties",
```
Comment (dianashk) [inline test_cases/wof_countries.json]: I think you meant `countries`

File: test_cases/wof_localities.json [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+          }
+        ]
+      }
+    },
+    {
+      "id": 3,
+      "status": "pass",
+      "user": "Stephen",
+      "in": {
+        "text": "Truth or Consequences, NM",
```
Comment (dianashk) [inline test_cases/wof_localities.json]: awesome city name!!

**Comment by trescube** [comment]
Find `localadmin` records that don't have the same names as `localities`

**Comment by dianashk** [comment]
Looking through the results now that integration is complete. Some findings...

#### `wof_counties#4`
 - [`text=Arzacq-Arraziguet`](http://pelias.github.io/compare/#/v1/search%3Fsources=wof&text=Arzacq-Arraziguet)
 - result looks as follows:
```
"properties": {
        "id": "102065177",
        "gid": "whosonfirst:county:102065177",
        "layer": "county",
        "source": "whosonfirst",
        "name": "Arzacq-Arraziguet",
        "confidence": 0.962,
        "country": "France",
        "country_id": "85633147",
        "country_a": "FRA",
        "region": "Pyrénées-Atlantiques",
        "region_id": "85683539",
        "county": "Arzacq-Arraziguet",
        "county_id": "102065177",
        "bounding_box": {
          "min_lat": 43.4194330071,
          "max_lat": 43.5962665102,
          "min_lon": -0.586993419958,
          "max_lon": -0.344441633102
        },
        "label": "Arzacq-Arraziguet, Pyrénées-Atlantiques, France"
}
```
 - expected result should be this [WOF record](https://whosonfirst.mapzen.com/spelunker/id/102065177/) which clearly has `macroregion` and `macrocounty` declared, however the data in our index is missing those fields. 

**Hierarchy**
> the continent of Europe
> the country of France
> the macroregion of Aquitaine
> the region of Pyrénées-Atlantiques
> the macrocounty of Pau
> the county of Arzacq-Arraziguet

**Comment by dianashk** [comment]
#### wof_countries#8

- [`text=BOL`](http://pelias.github.io/compare/#/v1/search%3Fsources=wof&text=BOL)
- the right result shows up second, but a place named Bol shows up first, because we score the match on the name field very high and don't explicitly check  against the `country_a` property. 
- code might need to be added to test any 3 letter values against country codes, and later airport codes and other possible 3 character-coded things.

**Pull request merged**